content/issues/4/unnatural-language/index.md

```diff
@@ -112,7 +105,7 @@ Another problem is the graphic variation common in early forms of Chinese, espec
           The medievalist Paul Zumthor referred to this idea as
           <a href="https://doi.org/10.1093/llc/fqx033">mouvance</a>, an 'interplay between variant readings and
           reworkings.' The concept shares some similarity with the idea of
-          <cite>generation loss</cite> in digital media: the phenomenon where
+          <cite class="book">generation loss</cite> in digital media: the phenomenon where
```

Comment:
oops, this isn't a book 🤦🏻  but i guess the same styles should apply. i did this to prevent the default styling for blockquote sources, which prepends a `- `, because i (mostly) use `<cite>` to semantically mark book titles in the figures/interludes

Reply:
ah, I was wondering. Could we scope the style to interludes or specific figures?

Reply:
yeah, that should work fine.